CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contribution guide
+
+Welcome! We are glad that you want to contribute to our project! 💖
+
+This project accepts contributions via Github pull requests.
+
+This document outlines the process to help get your contribution accepted.
+
+There are many ways to contribute:
+
+* Suggest [features](https://github.com/bouvet/nord-juice-shop/issues/new?assignees=&labels=%3Abulb%3A+feature+request&template=feature-request.md&title=)
+* Suggest [changes](https://github.com/bouvet/nord-juice-shop/issues/new?assignees=&labels=%3Awrench%3A+change&template=change-request.md&title=)
+* Report [bugs](https://github.com/bouvet/nord-juice-shop/issues/new?assignees=&labels=%3Abug%3A+bug&template=bug-report.md&title=)
+
+You can start by looking through the [good first issues](https://github.com/bouvet/nord-juice-shop/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+## Fork the repository
+
+In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr).
+
+Here's a quick guide:
+
+1. Create your own fork of the repository
+2. Clone the project to your machine
+3. To keep track of the original repository, add another remote named upstream
+```shell
+git remote add upstream git@github.com:bouvet/nord-juice-shop.git
+```
+4. Create a branch locally with a succinct but descriptive name and prefixed with change type.
+```shell
+git checkout -b feature/my-new-feature
```

Comment:
There's really no reason to require prefixing of branches as long as we don't use it for anything, like release please. Also, conventional commits use `feat` I think? not `feature`

Reply:
True! Imported this from another project I've worked on, so there may be some irrelevant entries. (Also: I guess the `feature` prefix is okay for branches, but the `feat` is for commits?)